jetty-server/src/main/java/org/eclipse/jetty/server/handler/BufferedResponseHandler.java

```diff
@@ -167,15 +172,28 @@ protected boolean isPathBufferable(String requestURI)
         return _paths.test(requestURI);
     }
 
-    private class BufferedInterceptor implements HttpOutput.Interceptor
+    protected BufferedInterceptor newBufferedInterceptor(HttpChannel httpChannel, Interceptor interceptor)
+    {
+        return new ArrayBufferedInterceptor(httpChannel, interceptor);
+    }
+
+    /**
+     * An {@link HttpOutput.Interceptor} which is created by {@link #newBufferedInterceptor(HttpChannel, Interceptor)}
+     * and is used by the implementation to buffer outgoing content.
+     */
+    protected interface BufferedInterceptor extends HttpOutput.Interceptor
+    {
+    }
+
+    private class ArrayBufferedInterceptor implements BufferedInterceptor
```

Comment:
Parameter `buffers` of `protected void commit(Queue<ByteBuffer> buffers, Callback callback)` is never used.
Either you don't pass it because you use the field, or you use it.
Please also use `Queue` methods rather than `Collection` methods, e.g. `offer()` instead of `add()`, `poll()` instead of `remove()`, etc.